backend/Pipfile

```diff
@@ -46,6 +46,8 @@ python-dateutil = "*"
 selenium = "*"
 webdriver-manager = "*"
 pre-commit = "*"
+profanity-check = "*"
+alt-profanity-check = "*"
```

Comment:
why do we need both? on alt-profanity-check's website it says: "Alt profanity check is a drop-in replacement of the profanity-check library for the not so well maintained", so it seems like just using alt-profanity-check should be fine.

Reply:
i had some weird installation errors without the alt-profanity-check, and it wouldn't run/pipfile wouldn't lock. online fixes said to do this

Reply:
oh wait jk getting rid of the first ill do that